src/game/shared/mapbase/vscript_singletons.cpp

```diff
@@ -1615,6 +1639,16 @@ class CScriptNetPropManager
 		}
 	}
 
+	bool GetPropBoolArray( HSCRIPT hEnt, const char *szProp, int index )
+	{
+		return (bool)GetPropIntArray( hEnt, szProp, index );
```

Comment:
Maybe this should return `GetPropIntArray == 1`. In `netpropmanager.cpp`, it returns `false` for invalid props, which would return `true` here. They also check int size but I don't think that's necessary.